lib/core/versions/latest/models/ServiceModel.ts

```diff
@@ -4,5 +4,5 @@
 export default interface ServiceModel {
   id: string;
   type: string;
-  serviceEndpoint: string | object;
+  serviceEndpoint: string | object | Array<string | object>;
```

Comment:
Probably no need for this change, array is an object.

Reply:
That's true, I just thought it might be better to understand if it's explicit. But I can also revert this change if you like.